packages/web/src/store/analytics/sagas.ts

```diff
@@ -5,6 +5,8 @@ import { take } from 'redux-saga/effects'
 
 import { env } from 'services/env'
 
+let prevPathname = ''
```

Comment:
hmm this scares me a bit, can we not use the react-router state to figure this out?

Reply:
NOTE: looked into this and decided that this was ok for this use-case bc connected-react-router does not easily expose path history